drivers/scheduler/k8s/k8s.go

```diff
@@ -1461,6 +1514,19 @@ func (k *K8s) Destroy(ctx *scheduler.Context, opts map[string]bool) error {
 		}
 	}
 
+	for _, spec := range ctx.App.SpecList {
+		t := func() (interface{}, bool, error) {
+			err := k.destroyVpsObjects(spec, ctx.App)
```

Comment:
we don't need a dedicated method `destroyVpsObjects`.

We would add a dedicated method where there is a group of objects in the same category that we need toc clean up.